### ISSUE TITLE
[FR] full keyboard navigation and shortcuts for grid data editing (including copying and pasting relational data)

### ISSUE THREAD

**Issue by Ray0m** [AppFlowy-IO/AppFlowy#8646]
### Description

Currently, it is impossible to use keyboard keys to navigate and edit databases in grid view. I would refer you to "typical" windows keyboard nav and shortcuts for most spreadsheet eidting as a reference point to see what is missing from appflowy but here is a general list of keyboard functionality I think we need in this app : 
using arrow keys to move from cell to cell 
crtl x to cut
ctrl c to copy
ctrl v to paste 
hold shift and arrow keys to create a group selection 
using mouse : click and hold to select multiple cells 
del key to delete cell content 
Importantly, this should also work for relational data. 

See video for demonstration of issue : https://www.youtube.com/watch?v=rKyoh00WFIk&t=1295s

personal importance level : 5/5 

### Impact

make editing of large data sets incredibly slow and frustrating.

**Comment by Ray0m** [comment]
no ETA on this one yet?
